dolphinscheduler-dao/src/main/resources/sql/upgrade/3.2.0_schema/mysql/dolphinscheduler_ddl.sql

```diff
@@ -119,4 +119,24 @@ d//
 
 delimiter ;
 CALL uc_dolphin_T_t_ds_task_instance_R_test_flag;
-DROP PROCEDURE uc_dolphin_T_t_ds_task_instance_R_test_flag;
\ No newline at end of file
+DROP PROCEDURE uc_dolphin_T_t_ds_task_instance_R_test_flag;
+
+-- add unique key to t_ds_process_definition_log
+drop PROCEDURE if EXISTS add_t_ds_process_definition_log_uk_uniq_idx_code_version;
+delimiter d//
+CREATE PROCEDURE add_t_ds_process_definition_log_uk_uniq_idx_code_version()
+BEGIN
```

Comment:
we have a discuss here, I think is a bug, and we should put it in version 3.1.1 instead of 3.2.0, WDTY @caishunfeng @EricGao888

Reply:
> we have a discuss here, I think is a bug, and we should put it in version 3.0.2 instead of 3.2.0, WDTY @caishunfeng @EricGao888 @SbloodyS

+1